README.md

```diff
@@ -63,7 +63,6 @@ The command removes all the Kubernetes objects associated with the chart and del
 | agent.constraintChecker.threshold | int | `3` | Number of failed checks before disabling task claim |
 | agent.containerSecurityContext | object | `{}` | Security Context policies for agent containers |
 | agent.customSecret | string | `""` | Name of the user provided secret containing resource class tokens. You can mix tokens from this secret and in the secret created from tokens specified in the resourceClasses section below Ref: https://circleci.com/docs/container-runner/#custom-secret  The tokens should be specified as secret key-value pairs of the form ResourceClass: Token The resource class name needs to match the names configured below exactly to match tokens to the correct configuration As Kubernetes does not allow / in secret keys, a period (.) should be substituted instead |
-| agent.dnsConfig.ndots | string | `"3"` | Minimum number of dots in a domain name to prevent name lookups using the DNS suffix search list |
```

Comment:
This wasn't properly added to the values file, so this fails the automated readme generation/validation: https://app.circleci.com/pipelines/github/CircleCI-Public/container-runner-helm-chart/419/workflows/71f26521-4ef5-4a50-9134-22f3bb0ed823/jobs/1240?invite=true#step-104-0_119

Will attempt a fix in a different PR